simapp/ante.go

```diff
@@ -31,17 +31,18 @@ func NewAnteHandler(options HandlerOptions) (sdk.AnteHandler, error) {
 		return nil, errors.New("sign mode handler is required for ante builder")
 	}
 
+	// HV2: commented out some decorators according to heimdall's business logic (see x/auth/ante/ante.go)
 	anteDecorators := []sdk.AnteDecorator{
 		ante.NewSetUpContextDecorator(), // outermost AnteDecorator. SetUpContext must be called first
 		circuitante.NewCircuitBreakerDecorator(options.CircuitKeeper),
 		ante.NewExtensionOptionsDecorator(options.ExtensionOptionChecker),
 		ante.NewValidateBasicDecorator(),
-		ante.NewTxTimeoutHeightDecorator(),
+		// ante.NewTxTimeoutHeightDecorator(),
```

Comment:
With the purpose of only preserving only the `heimdall` changes, I removed everything which was not there.
Maybe we can investigate this functionality and enable it afterwards. Wdyt? In case, I'll create a JIRA task.